evap/evaluation/migrations/0076_add_semester_short_names.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 2.0.2 on 2018-02-27 21:19
+
+from django.db import migrations, models
+
+
+def populate_semester_short_names(apps, schema_editor):
```

Comment:
do we want to manually remove the `schema_editor` here?

Reply:
i've done it in this PR but can't see us enforcing that... i don't know. we'll see.